contracts/bonding/BondingManager.sol

```diff
@@ -101,10 +101,10 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
     uint256 public nextRoundTotalActiveStake;
 
     // Check if sender is TicketBroker
-    modifier onlyTicketBroker() {
+    modifier onlyJobsManager() {
```

Comment:
Perhaps we should consider registering `TicketBroker` as both `TicketBroker` and `JobsManager` and revert the name changes here.

This would reduce the amount of possible confusion (also with the revert reason). Additionally since `updateTranscoderWithFees` where this modifier is used does not involve any direct `Minter` logic, so I would keep the scope of reverting modifier name changes to the `Minter` contract. WDYT?

Reply:
Agreed that it feels clearer for other contracts besides the Minter that *are* aware of the TicketBroker contract ID to use the `onlyTicketBroker` modifier. Fixed in c7d0ee4